src/Datadog.Trace.BenchmarkDotNet/CIEnvironmentValues.cs

```diff
@@ -0,0 +1,218 @@
+using System;
+using System.Text.RegularExpressions;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
+using Datadog.Trace.Vendors.Serilog;
+
+namespace Datadog.Trace.BenchmarkDotNet
+{
+    internal static class CIEnvironmentValues
+    {
+        private static readonly Regex BranchRegex = new Regex(@"^refs\/heads\/(.*)|refs\/(.*)$", RegexOptions.Compiled);
+        private static readonly ILogger Logger = DatadogLogging.GetLogger(typeof(CIEnvironmentValues));
+
+        static CIEnvironmentValues()
```

Comment:
For clarity, I think each branch should be extracted into a separate method

(could be local functions if you're afraid of polluting the class)


```
if (EnvironmentHelpers.GetEnvironmentVariable("TRAVIS") != null)
{
    SetupTravisEnvironment();
}
else if (...)
```

Reply:
Ok, I'll do the change.

Reply:
I remove this file, to use the shared project already in master that contains this class.

Reply:
With the https://github.com/DataDog/dd-trace-dotnet/pull/811 merge, this recommendation is already on master.